tools/mqtt-bench/bench.go

```diff
@@ -148,12 +152,14 @@ func Benchmark(cfg Config) {
 
 	wg.Wait()
 
+	start := time.Now()
 	// Publishers
 	for i := 0; i < cfg.Test.Pubs; i++ {
-		mfConn := mf.Conns[i%n]
+		mfConn := mf.Channels[i%n]
```

Comment:
Using `mfChann` instead of `mfConn` makes more sense.

Reply:
Agreed